submissions/cuteman18/popup/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML & CSS practice</title>
+    <link rel="stylesheet" type="text/css" href="app.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  </head>
+  <body>
+    <nav>
+      <ul class="navList">
+        <li class="inscription"><a href="#" class="ref">About</a></li>
+        <li class="inscriptionDeviate"><a href="https://youtu.be/xdYYN-4ttDg" class="ref" target="_blank">Hobby</a></li>
+        <li class="inscription"><a href="#" class="ref">Mail</a></li>
+        <li class="inscription"><a href="#" class="ref">Images</a></li>
+        <li class="inscription">
+          <input type="checkbox" id="drop" class="dropDown"/><label for="drop" class="holder"></label>
+          <div class="menu">
+            <ul>
+              <li class="icons"><a href="#" class="ref2"><span class="icon"><img class="imgStyle" src="images/calendar.png" alt="calendar"/></span><span class="iconText">calendar</span></a></li>
+              <li class="icons"><a href="#" class="ref2"><span class="icon"><img class="imgStyle" src="images/mail.png" alt="mail"/></span><span class="iconText">mail</span></a></li>
+              <li class="icons"><a href="#" class="ref2"><span class="icon"><img class="imgStyle" src="images/news.png" alt="news"/></span><span class="iconText">news</span></a></li>
+              <li class="icons"><a href="#" class="ref2"><span class="icon"><img class="imgStyle" src="images/maps.png" alt="maps"/></span><span class="iconText">maps</span></a></li>
+              <li class="icons"><a href="#" class="ref2"><span class="icon"><img class="imgStyle" src="images/files.png" alt="files"/></span><span class="iconText">files</span></a></li>
+              <li class="icons"><a href="#" class="ref2"><span class="icon"><img class="imgStyle" src="images/contacts.png" alt="contacts"/></span><span class="iconText">contacts</span></a></li>
+              <li class="icons"><a href="#" class="ref2"><span class="icon"><img class="imgStyle" src="images/photos.png" alt="photos"/></span><span class="iconText">photo</span></a></li>
+              <li class="icons"><a href="#" class="ref2"><span class="icon"><img class="imgStyle" src="images/videos.png" alt="videos"/></span><span class="iconText">video</span></a></li>
+              <li class="icons"><a href="#" class="ref2"><span class="icon"><img class="imgStyle" src="images/search.png" alt="search"/></span><span class="iconText">search</span></a></li>
+              <hr>
+              <input type="checkbox" class="button" id="extra"/><label for="extra" class="more">more</label>
```

Comment:
This button still without styles on focus state.
I wrote about this in [PR213](https://github.com/kottans/frontend-2021-homeworks/pull/213) and [PR412](https://github.com/kottans/frontend-2021-homeworks/pull/412)

Reply:
@A-Ostrovnyy  Hello Alexander, sorry I thought you were asking about adding more focus on icons.    I added focus to the button (lines 85-88 in app.css), please review.